doc/nrf/config_and_build/dfu/nrf70_fw_patch_update.rst

```diff
@@ -0,0 +1,207 @@
+    The example provided is for illustrative purposes and may need to be adjusted to fit the actual configuration file format and syntax required by the Partition Manager in use.
+
+Configuring build system
+------------------------
+
+To enable the DFU procedure for the nRF70 Series firmware patch, complete the following steps depending on the platform:
+
+* For the nRF5340 DK without the network core:
+
+    1. Set the :kconfig:option:`CONFIG_NRF_WIFI_FW_PATCH_DFU`` Kconfig option to ``y``.
+    #. Set the :kconfig:option:`CONFIG_UPDATEABLE_IMAGE_NUMBER`` Kconfig option to ``2``.
+    #. For the MCUBoot child image, set the :kconfig:option:`CONFIG_UPDATEABLE_IMAGE_NUMBER`` Kconfig option to ``2``.
+
+* For the nRF5340 DK with the network core:
+
+    1. Set the :kconfig:option:`CONFIG_NRF_WIFI_FW_PATCH_DFU`` Kconfig option to ``y``.
+    #. Set the :kconfig:option:`CONFIG_UPDATEABLE_IMAGE_NUMBER`` Kconfig option to ``3``.
+    #. For the MCUBoot child image, set the :kconfig:option:`CONFIG_UPDATEABLE_IMAGE_NUMBER`` Kconfig option to ``3``.
+
+For example, to build the sample with the DFU procedure for the nRF70 Series firmware patch on the nRF5340 DK platform, which includes the network core image, run the following commands:
+
+.. tabs::
+
+   .. group-tab:: West
+
+        .. code-block:: console
+
+            west build -d nrf5340dk_nrf5340_cpuapp -d -- -DSHIELD=nrf7002ek -DCONFIG_PARTITION_MANAGER_ENABLED=y -DCONFIG_NRF_WIFI_PATCHES_EXT_FLASH_STORE=y -DCONFIG_NRF_WIFI_FW_PATCH_DFU=y -DCONFIG_UPDATEABLE_IMAGE_NUMBER=3 -Dmcuboot_CONFIG_UPDATEABLE_IMAGE_NUMBER=3
+
+   .. group-tab:: CMake
+
+        .. code-block:: console
+
+            cmake -GNinja -Bbuild -DBOARD=nrf5340dk_nrf5340_cpuapp -DSHIELD=nrf7002ek -DCONFIG_PARTITION_MANAGER_ENABLED=y -DCONFIG_NRF_WIFI_PATCHES_EXT_FLASH_STORE=y -DCONFIG_NRF_WIFI_FW_PATCH_DFU=y -DCONFIG_UPDATEABLE_IMAGE_NUMBER=3 -Dmcuboot_CONFIG_UPDATEABLE_IMAGE_NUMBER=3 sample
+            ninja -C build
+
+   .. group-tab:: nRF Connect for VS Code
+
+        1. When `building an application <How to build an application_>`_ as described in the |nRFVSC| documentation, follow the steps for setting up the build configuration.
+        #. In the Add Build Configuration screen, click the Add argument button under the Extra CMake argument section.
+        #. Add the following Kconfig options:
+
+        .. code-block:: console
+
+            -- -DSHIELD=nrf7002ek -DCONFIG_PARTITION_MANAGER_ENABLED=y -DCONFIG_NRF_WIFI_PATCHES_EXT_FLASH_STORE=y -DCONFIG_NRF_WIFI_FW_PATCH_DFU=y -DCONFIG_UPDATEABLE_IMAGE_NUMBER=3 -Dmcuboot_CONFIG_UPDATEABLE_IMAGE_NUMBER=3
+
+If you want to use the :ref:`ug_multi_image` feature, you need to set the :kconfig:option:`CONFIG_DFU_MULTI_IMAGE_MAX_IMAGE_COUNT` Kconfig option to one of the following values:
+
+* For the nRF5340 DK without the network core: ``2``
+* For the nRF5340 DK with the network core: ``3``
+
```

Comment:
as above

Reply:
It can be added as a follow-up after NCS 2.6.0, as @nvlsianpu mentioned above.